{
  "repository": {
    "name": "Respekto/posesor-backend",
    "githubId": 64332602,
    "url": "https://github.com/Respekto/posesor-backend",
    "isFork": false,
    "original": {
      "name": "",
      "githubId": 0,
      "url": ""
    },
    "isPullRequest": true,
    "pullRequestId": 29
  },
  "builds": {
    "buggyBuild": {
      "id": 411008802,
      "url": "http://travis-ci.org/Respekto/posesor-backend/builds/411008802",
      "date": "Aug 1, 2018 11:30:56 PM"
    }
  },
  "reproductionBuggyBuild": {
    "reproductionDateBeginning": "Aug 1, 2018 11:31:09 PM",
    "reproductionDateEnd": "Aug 1, 2018 11:43:54 PM",
    "totalDuration": 759,
    "processDurations": {
      "cloning": {
        "nbSteps": 1,
        "totalDuration": 5,
        "stepNames": [
          "CloneRepository"
        ],
        "stepDurations": [
          5
        ]
      },
      "building": {
        "nbSteps": 2,
        "totalDuration": 123,
        "stepNames": [
          "CheckoutBuggyBuild",
          "BuildProject"
        ],
        "stepDurations": [
          1,
          122
        ]
      },
      "testing": {
        "nbSteps": 1,
        "totalDuration": 340,
        "stepNames": [
          "TestProject"
        ],
        "stepDurations": [
          340
        ]
      }
    },
    "projectRootPomPath": "./workspace/Respekto/posesor-backend/411008802/pom.xml"
  },
  "patchDiff": {
    "files": {
      "numberAdded": 0,
      "numberChanged": 0,
      "numberDeleted": 0
    },
    "lines": {
      "numberAdded": 0,
      "numberDeleted": 0
    }
  },
  "commits": {
    "buggyBuild": {
      "repoName": "Respekto/posesor-backend",
      "branchName": "master",
      "sha": "d2739a4d3f16fb8898c86c93dac05d0eb2b20e7a",
      "url": "http://github.com/Respekto/posesor-backend/commit/d2739a4d3f16fb8898c86c93dac05d0eb2b20e7a",
      "date": "Aug 1, 2018 11:23:34 PM"
    },
    "buggyBuildForkRepo": {
      "repoName": "Respekto/posesor-backend",
      "branchName": "issue-28",
      "sha": "8a8b65455a3425ce6c3437a598e5fe0f6d92c466",
      "url": "https://github.com/Respekto/posesor-backend/commit/8a8b65455a3425ce6c3437a598e5fe0f6d92c466",
      "date": "Aug 1, 2018 11:23:34 PM"
    },
    "buggyBuildBaseRepo": {
      "repoName": "Respekto/posesor-backend",
      "branchName": "master",
      "sha": "8df6474347614b3421799f7e3434b945d92e3155",
      "url": "https://github.com/Respekto/posesor-backend/commit/8df6474347614b3421799f7e3434b945d92e3155",
      "date": "Jul 27, 2018 11:34:49 PM"
    }
  },
  "projectMetrics": {
    "numberModules": 5,
    "numberSourceFiles": 125,
    "numberTestFiles": 31,
    "numberLibrariesFailingModule": 108
  },
  "tests": {
    "failingModule": "/root/workspace/Respekto/posesor-backend/411008802/posesor-domain",
    "overallMetrics": {
      "numberRunning": 11,
      "numberPassing": 6,
      "numberFailing": 0,
      "numberErroring": 5,
      "numberSkipping": 0,
      "failures": [
        {
          "failureName": "org.junit.runners.model.TestTimedOutException",
          "isError": true,
          "occurrences": 1
        },
        {
          "failureName": "java.util.concurrent.ExecutionException",
          "isError": true,
          "occurrences": 4
        }
      ]
    },
    "failingClasses": [
      {
        "testClass": "net.posesor.PaymentTitleSearchShould",
        "numberRunning": 5,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 5,
        "numberSkipping": 0
      }
    ],
    "failureDetails": [
      {
        "testClass": "net.posesor.PaymentTitleSearchShould",
        "testMethod": "includeChangedTitle",
        "failureName": "java.util.concurrent.ExecutionException",
        "detail": "org.springframework.dao.DataAccessResourceFailureException: Timed out after 30000 ms while waiting for a server that matches ReadPreferenceServerSelector{readPreference\u003dprimary}. Client view of cluster state is {type\u003dUNKNOWN, servers\u003d[{address\u003dlocalhost:27017, type\u003dUNKNOWN, state\u003dCONNECTING, exception\u003d{com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused (Connection refused)}}]; nested exception is com.mongodb.MongoTimeoutException: Timed out after 30000 ms while waiting for a server that matches ReadPreferenceServerSelector{readPreference\u003dprimary}. Client view of cluster state is {type\u003dUNKNOWN, servers\u003d[{address\u003dlocalhost:27017, type\u003dUNKNOWN, state\u003dCONNECTING, exception\u003d{com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused (Connection refused)}}]",
        "isError": true
      },
      {
        "testClass": "net.posesor.PaymentTitleSearchShould",
        "testMethod": "filter",
        "failureName": "java.util.concurrent.ExecutionException",
        "detail": "org.springframework.dao.DataAccessResourceFailureException: Timed out after 30000 ms while waiting for a server that matches ReadPreferenceServerSelector{readPreference\u003dprimary}. Client view of cluster state is {type\u003dUNKNOWN, servers\u003d[{address\u003dlocalhost:27017, type\u003dUNKNOWN, state\u003dCONNECTING, exception\u003d{com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused (Connection refused)}}]; nested exception is com.mongodb.MongoTimeoutException: Timed out after 30000 ms while waiting for a server that matches ReadPreferenceServerSelector{readPreference\u003dprimary}. Client view of cluster state is {type\u003dUNKNOWN, servers\u003d[{address\u003dlocalhost:27017, type\u003dUNKNOWN, state\u003dCONNECTING, exception\u003d{com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused (Connection refused)}}]",
        "isError": true
      },
      {
        "testClass": "net.posesor.PaymentTitleSearchShould",
        "testMethod": "includeChargeTitleAsPaymentTitle",
        "failureName": "java.util.concurrent.ExecutionException",
        "detail": "org.springframework.dao.DataAccessResourceFailureException: Timed out after 30000 ms while waiting for a server that matches ReadPreferenceServerSelector{readPreference\u003dprimary}. Client view of cluster state is {type\u003dUNKNOWN, servers\u003d[{address\u003dlocalhost:27017, type\u003dUNKNOWN, state\u003dCONNECTING, exception\u003d{com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused (Connection refused)}}]; nested exception is com.mongodb.MongoTimeoutException: Timed out after 30000 ms while waiting for a server that matches ReadPreferenceServerSelector{readPreference\u003dprimary}. Client view of cluster state is {type\u003dUNKNOWN, servers\u003d[{address\u003dlocalhost:27017, type\u003dUNKNOWN, state\u003dCONNECTING, exception\u003d{com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused (Connection refused)}}]",
        "isError": true
      },
      {
        "testClass": "net.posesor.PaymentTitleSearchShould",
        "testMethod": "removeNotUsedPaymentTitle",
        "failureName": "java.util.concurrent.ExecutionException",
        "detail": "org.springframework.dao.DataAccessResourceFailureException: Timed out after 30000 ms while waiting for a server that matches ReadPreferenceServerSelector{readPreference\u003dprimary}. Client view of cluster state is {type\u003dUNKNOWN, servers\u003d[{address\u003dlocalhost:27017, type\u003dUNKNOWN, state\u003dCONNECTING, exception\u003d{com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused (Connection refused)}}]; nested exception is com.mongodb.MongoTimeoutException: Timed out after 30000 ms while waiting for a server that matches ReadPreferenceServerSelector{readPreference\u003dprimary}. Client view of cluster state is {type\u003dUNKNOWN, servers\u003d[{address\u003dlocalhost:27017, type\u003dUNKNOWN, state\u003dCONNECTING, exception\u003d{com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused (Connection refused)}}]",
        "isError": true
      },
      {
        "testClass": "net.posesor.PaymentTitleSearchShould",
        "testMethod": "keepPaymentTitleUsedMultipleTimes",
        "failureName": "org.junit.runners.model.TestTimedOutException",
        "detail": "test timed out after 1000 milliseconds",
        "isError": true
      }
    ]
  },
  "type": "only_fail"
}